daemon/src/taker_cfd.rs

```diff
@@ -198,61 +187,78 @@ where
             .await?;
         Ok(())
     }
-}
 
-#[xtra_productivity]
-impl<O, M, W> Actor<O, M, W> {
-    async fn handle_propose_settlement(&mut self, msg: ProposeSettlement) -> Result<()> {
+    async fn handle_propose_settlement(
+        &mut self,
+        msg: ProposeSettlement,
+        ctx: &mut xtra::Context<Self>,
+    ) -> Result<()> {
         let ProposeSettlement {
             order_id,
             current_price,
         } = msg;
 
+        let disconnected = self
```

Comment:
what does "disconnected" mean in this context?

Reply:
It is an instance of `address_map::Disconnected`. Do you have an idea for a better name?

Reply:
hard to think of a better one inded. If I find one, I'll propose something!